### PR TITLE
Ignore duplicate events

### PR DESCRIPTION
**Pull request by sarayourfriend**

### CODE REVIEW

File: seattleblmcalendar.rss [hunk 1]
```diff
@@ -0,0 +1,515 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AetherUnbound) [inline seattleblmcalendar.rss]: I don't think we want to include these two files

Comment (sarayourfriend) [reply seattleblmcalendar.rss]: Definitely not, removed em!

File: config.py.example [hunk 2]
```diff
@@ -23,6 +23,9 @@ TWITTER_ACCESS_SECRET='Insert-Your-Twitter-Access-Token-Secret'
 # (such as calendar title and description). Subsequent calendars 
 # contribute only events.
 CALENDAR_IDS=[
+    "3rvtf5quvg3j7cm87iu1t4du20@group.calendar.google.com",
+    "67ut0oii4uma5rfsfiic7gbqtg@group.calendar.google.com",
+    "4khfuvsaf6r9vh6mkqif7gm5o0@group.calendar.google.com",
     "bmu9a32fkrmvmi8mu1hegp45f4@group.calendar.google.com",
```
Comment (sarayourfriend) [inline config.py.example:29]: Apparently I had misinterpreted the calendar ID for the mutual aid calendar as a collective calendar ID for all four calendars, but indeed we need to include them separately.

File: gcal_helpers/helpers.py [hunk 3]
```diff
@@ -687,6 +703,8 @@ def generate_rss(cal_dict):
 
     time_now = get_time_now()
 
+    cal_dict['items'] = sort_by_date(filter_duplicate_guids(cal_dict['items']))
```
Comment (AetherUnbound) [inline gcal_helpers/helpers.py:719]: Do you mind adding a comment just above this describing what/why this is being done?

**Review by AetherUnbound — APPROVED** [review]
Just one thought, nice work!

**Pull request merged**